src/lib/es2023.array.d.ts

```diff
@@ -101,12 +101,12 @@ interface ReadonlyArray<T> {
     ): number;
 
     /**
-     * Copies the array and returns the copied array with all of its elements reversed.
+     * Returns a copy of an array with its elements reversed.
      */
     toReversed(): T[];
 
     /**
-     * Copies and sorts the array.
+     * Returns a copy of an array with its elements sorted.

```

Comment:
This is an improvement, but I would do

```suggestion
     * Returns a sorted copy of an array.
```